sdk/servicebus/service-bus/src/util/parseUrl.browser.ts

```diff
@@ -10,6 +10,8 @@
  * @param {string} rawUrl
  * @returns
  */
-export const parseURL = (rawUrl: string) => {
+export const parseURL = (rawUrl: string): any => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
an ignore for an ignore... maybe we should ditch the ban-ts-ignore rule if we are going to need to use ts-ignore?

Reply:
@willmtemple ^